appinventor/blocklyeditor/src/msg/zh_tw/_messages.js

```diff
@@ -120,200 +122,200 @@ Blockly.Msg.zh.switch_language_to_chinese_tw = {
 
     Blockly.Msg.LANG_CONTROLS_IF_ELSEIF_TITLE_ELSEIF = '否則，如果';
-    Blockly.Msg.LANG_CONTROLS_IF_ELSEIF_TOOLTIP = '為「如果」語句塊增設條件';
+    Blockly.Msg.LANG_CONTROLS_IF_ELSEIF_TOOLTIP = '為「如果」程式方塊增設條件';
 
     Blockly.Msg.LANG_CONTROLS_IF_ELSE_TITLE_ELSE = '否則';
-    Blockly.Msg.LANG_CONTROLS_IF_ELSE_TOOLTIP = '設最終條件，當所有條件均不滿足時則執行最終條件';
+    Blockly.Msg.LANG_CONTROLS_IF_ELSE_TOOLTIP = '最終條件，當所有條件均不滿足時執行本項';
 
     Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_HELPURL = 'http://appinventor.mit.edu/explore/ai2/support/blocks/control#while';
-    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TITLE_REPEAT = '重複';
+    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TITLE_REPEAT = '當滿足條件…執行';
     Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_INPUT_DO = '執行';
-    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_OPERATOR_WHILE = '只要';
+    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_OPERATOR_WHILE = '當';
     Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_OPERATOR_UNTIL = '直到';
-    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_WHILE = '只要值為真，就重複執行相關語句';
-    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL = '只要值為假，就重複執行相關語句';
-    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_1 = '只要條件為真，就執行「執行」區域所包含的語句塊';
+    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_WHILE = '只要值為真，就重複執行「當滿足條件…執行」內的程式方塊';
+    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_UNTIL = '只要值為假，就重複執行「直到滿足條件…執行」內的程式方塊';
+    Blockly.Msg.LANG_CONTROLS_WHILEUNTIL_TOOLTIP_1 = '只要條件為真，就執行「當滿足條件…執行」內的程式方塊';
 
     Blockly.Msg.LANG_CONTROLS_FOR_HELPURL = '';
-    Blockly.Msg.LANG_CONTROLS_FOR_INPUT_WITH = '循環取數到';
-    Blockly.Msg.LANG_CONTROLS_FOR_INPUT_VAR = 'x';
+    Blockly.Msg.LANG_CONTROLS_FOR_INPUT_WITH = '依序取數到';
+    Blockly.Msg.LANG_CONTROLS_FOR_INPUT_VAR = '數字';
     Blockly.Msg.LANG_CONTROLS_FOR_INPUT_FROM = '範圍從';
     Blockly.Msg.LANG_CONTROLS_FOR_INPUT_TO = '到';
     Blockly.Msg.LANG_CONTROLS_FOR_INPUT_DO = '執行';
 
     Blockly.Msg.LANG_CONTROLS_FOR_TOOLTIP = '從一個數字開始取數，到另一個數結束。\n' +
-        '每取一個數，都將其值賦給\n' +
-        '變數 "%1"，並執行語句塊。';
+        '每取一個數，都將其值賦值予\n' +
+        '變數 "%1"，並執行程式方塊。';
 
     Blockly.Msg.LANG_CONTROLS_FORRANGE_HELPURL = 'http://appinventor.mit.edu/explore/ai2/support/blocks/control#forrange';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_ITEM = '對於任意';
-    Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_VAR = '變數名';
+    Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_VAR = '數字';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_START = '範圍從';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_END = '到';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_STEP = '每次增加';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_DO = '執行';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_COLLAPSED_TEXT = '對一定範圍內的數字';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_COLLAPSED_PREFIX = '對於 ';
     Blockly.Msg.LANG_CONTROLS_FORRANGE_INPUT_COLLAPSED_SUFFIX = ' 範圍內的';
-    Blockly.Msg.LANG_CONTROLS_FORRANGE_TOOLTIP = '按指定範圍和增量循環取值，'
+    Blockly.Msg.LANG_CONTROLS_FORRANGE_TOOLTIP = '按指定範圍和增量依序取值';
 	+ '每次循環均將數值賦予指定變數，'
-	+ '並運行「執行」區域所包含的代碼塊';
+	+ '並執行「執行」區段中的程式方塊'';
```

Comment:
@cavedunissin There is an extra ' here that is preventing this change from compiling. Please fix the error and ensure that the sources compile.